index.css

```diff
@@ -0,0 +1,204 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.Header {
+    margin-top: -40px;
+    width: 100%;
+    color: white;
+    background-color: #1f2937;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 25rem;
+
+}
+.header-logo-text{
+    font-size: 24px;
+    color: #f9faf8;
+}
+
+nav {
+
+    display: flex;
+    justify-content: space-between;
+    color: white;
+}
+
+ul {
+    list-style: none;
+    display: flex;
+
+}
+
+.nav-text {
+    color: white;
+    margin-left: 10px;
+    outline: none;
+    text-decoration: none;
+}
+
+.mid-section {
+    margin-top: 4rem;
+    display: flex;
+}
+
+.mainText{
+    font-size: 48px;
+    font-weight: bold;
+}
+
+.secondary-text{
+    font-size: 18px;
+    color:  #e5e7eb;
+}
+
+.mainText-section{
+    width: 40%;
+    margin-right: 2rem;
+}
+
+
+.img-placeholder{
+    background-color: #565a5e;
+    width: 50%;
+    height: 10rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.signup-button {
+    cursor: pointer;
+    margin-top: 8px;
+    color: white;
+    padding: 4px 10px;
+    background-color: #3882f6;
+    border-radius: 5px;
+    border: none;
+}
+
```

Comment:
one line diff is enough when you start a new entity remove the extra line from here as well